metricbeat/module/ceph/df/df.go

```diff
@@ -0,0 +1,44 @@
+package df
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/module/ceph"
+)
+
+func init() {
+	if err := mb.Registry.AddMetricSet("ceph", "df", New); err != nil {
+		panic(err)
+	}
+}
+
+type MetricSet struct {
+	mb.BaseMetricSet
+	cfg *ceph.CephConfig
+}
+
+func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
+
+	config := ceph.Config{}
```

Comment:
All the Ceph metricsets should be marked experimental first. We do this for all metricsets for the first release. See other metricsets for examples.